.nvmrc

```diff
@@ -1 +1 @@
-16.18.1
\ No newline at end of file
+18.12.1
```

Comment:
Need to also change the Github Actions YML files to see if the code builds and runs with the new node version.

Reply:
Done.